src/navbar/navbar-action-buttons.styles.tsx

```diff
@@ -18,6 +18,17 @@ export const Wrapper = styled.ul`
 `;
 
 export const MobileWrapper = styled.ul`
+    display: flex;
+    list-style: none;
+    margin-left: 4rem;
+    flex-shrink: 0;
+
+    ${MediaQuery.MinWidth.tablet} {
```

Comment:
try to use `MediaQuery.MaxWidth` instead because there's this 1px point where the media queries don't overlap resulting in the button being shown in the navbar